class.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Thank You"
+layout: single
+permalink: /class
+
+---
+We're sure you are excited about that class. In the meantime, you might want to find out [more](/about) about All Hands Active or even become a [member](/membership). 
+
+### Stay informed about all things All Hands Active! 
+Sign up for our email list or follow us on social media (at the bottom of the page).
+
+<!-- Begin Mailchimp Signup Form -->
+<link href="//cdn-images.mailchimp.com/embedcode/horizontal-slim-10_7.css" rel="stylesheet" type="text/css">
+<style type="text/css">
+  #mc_embed_signup{background:#eee; clear:left; font:14px Helvetica,Arial,sans-serif; width:100%;}
+  form{background:#eee;}
+</style>
+<div id="mc_embed_signup">
+<form action="https://allhandsactive.us4.list-manage.com/subscribe/post?u=0497c04d5751340fb683e3d14&amp;id=5612521156" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
+    <div id="mc_embed_signup_scroll">
+      <input type="email" value="" name="EMAIL" class="email" id="mce-EMAIL" placeholder="email address" style="height: 3em;padding: .5em 1em;font-size: 1.2em;font-weight: bold;" required>
+      <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
+      <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_0497c04d5751340fb683e3d14_5612521156" tabindex="-1" value=""></div>
+      <div class="clear"><input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button" style="background:#0092CA; color:#fff; padding: 0.5em 1em; height: 3em; font-size: 1.2em; font-weight:bold;"></div>
+    </div>
+  </form>
+</div>
+
+### Find another class
```

Comment:
Should be an H2

Reply:
> * Can you confirm the image you chose isn't encumbered (it's public domain or creative commons)?
> * What would cause someone to end up at /cancel?
> * What would cause someone to end up at /class?

license is free. see it at unsplash https://unsplash.com/license

flow for /cancel is as follows:
sign up for a class
go to /pay
click on donate
click cancel instead of completing donation
directed to /cancel

flow for /class is as follows:
sign up for a class
go to /pay
click on donate
make a payment
directed to /class